strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-maven-metadata-api/src/main/java/org/carlspring/strongbox/storage/metadata/VersionCollector.java

```diff
@@ -1,10 +1,10 @@
 package org.carlspring.strongbox.storage.metadata;
 
-import org.carlspring.strongbox.io.filters.ArtifactVersionDirectoryFilter;
-import org.carlspring.strongbox.storage.metadata.comparators.MetadataVersionComparator;
-import org.carlspring.strongbox.storage.metadata.comparators.SnapshotVersionComparator;
-import org.carlspring.strongbox.storage.metadata.versions.MetadataVersion;
-import org.carlspring.strongbox.storage.metadata.visitors.ArtifactVersionDirectoryVisitor;
+import org.carlspring.strongbox.storage.metadata.maven.ArtifactVersionDirectoryFilter;
```

Comment:
Hi!
This is not what I had in mind. The sub-packages need to be moved (preserved) as well.

Reply:
Oh right! Read it wrongly, changed it